### ISSUE TITLE
scrollTo has momentum?

### ISSUE THREAD

**Issue by Fergie77** [darkroomengineering/lenis#171]
I've been trying to use lenis to scroll to elements similarly to how fullpage.js works. Basically I have an element on a page which is full height and I'd like that once the user scrolls past a certain point, lenis scrolls to make the element fill the page. 

The problem I'm having is that lenis seems to have a bit of extra momentum from the scroll itself, so overshoots slightly depending on how hard the scroll was. Is there a way I can remove the momentum just before the scrollTo function?

**Comment by clementroche** [comment]
Hi, i don't know what is fullpage.js can you provide an example ?
I think you need [Lenis CSS scroll-snap plugin ](https://github.com/funkhaus/lenis-scroll-snap)

**Comment by Fergie77** [comment]
> Hi, i don't know what is fullpage.js can you provide an example ? I think you need [Lenis CSS scroll-snap plugin ](https://github.com/funkhaus/lenis-scroll-snap)

You can find it here https://alvarotrigo.com/fullPage/

I've tried using that plugin itself but it doesn't play very well with lenis, so have avoided it. However the functionality of having a section perfectly framed is what I'm looking for!

In the site I'm working on, if you scroll down to the pink slider you should hopefully be able to see what I mean.

https://buttermilk-agency.webflow.io/brand-fans

Basically if you scroll on to the trigger which activates the scrollTo function very slowly, it'll fill the screen. However if you flick strongly (I've only tested on a trackpad), you end up further past the element. I've attached two screenshots of those two cases

<img width="1512" alt="image" src="https://github.com/studio-freight/lenis/assets/48885644/4ad09576-9c73-48ba-9d2d-aa86cdf7e4da">

<img width="1511" alt="image" src="https://github.com/studio-freight/lenis/assets/48885644/2da537ce-a5d6-47cc-bd0b-4684c35b7fbe">

**Comment by Fergie77** [comment]
Also I should note that I'm only looking for this full page functionality on this one element of the page. Everything else should scroll normally.

**Comment by Fergie77** [comment]
<img width="612" alt="image" src="https://github.com/studio-freight/lenis/assets/48885644/c16485d3-9a6b-49f1-a20b-3cad6eff5806">

Currently using this code to have it scroll to that section when the GSAP trigger happens.
